gammapy/image/tests/test_maps.py

```diff
@@ -45,6 +45,10 @@ def test_coordinates(self):
         assert_allclose(coordinates[0][100, 100].degree, 0.01)
         assert_allclose(coordinates[1][100, 100].degree, 0.01)
 
+    def test_solid_angle(self):
```

Comment:
That's strange...it shouldn't be negative and it isn't the case for the test files I used. But let's change to make it return safe results...